Sources/PointFree/Account/Invite.swift

```diff
@@ -46,7 +46,18 @@ let revokeInviteMiddleware: Middleware<StatusLineOpen, ResponseEnded, Tuple2<Tea
     <| { conn in
       Current.database.deleteTeamInvite(get1(conn.data).id)
         .run
-        .flatMap(const(conn |> redirect(to: path(to: .account(.index)))))
+        .flatMap(
+          const(
+            conn
+              |> redirect(
+                to: path(to: .account(.index)),
+                headersMiddleware: flash(
+                  .notice,
+                  "Invite to \(get1(conn.data).email.rawValue) has been revoked."
```

Comment:
unrelated to this PR but I realized I dont show a flash when revoking an invite, so adding that.